### PR TITLE
remove my little pony referneces 

### PR DESCRIPTION
**Pull request by boodaliboo**
### Intent of your Pull Request
ponys bad also thanks to hippie for this pr which i stole some text from xoxoxo https://github.com/HippieStation/HippieStation/pull/12435
#### Changelog

:cl:    
tweak: changed descriptions
/:cl:

### CODE REVIEW

**Comment by MegaEmpirical** [comment]
Is there a point to this?

**Comment by Hopekz** [comment]
I don't like these new descriptions but I don't see this making a negative impact on our game so i'm neutral. 

Old descriptions that you are trying to replace are short and sweet per the original author's intent.
Hopefully Hippie has something better to port.

You kind of just replaced short and to the point descriptions with monologue on the item to try to achieve the same effect as the original description.

**Comment by boodaliboo** [comment]
I can shorten it but I just don’t want references to a furry bait show

**Comment by MegaEmpirical** [comment]
I think your reasoning for this is dumb and without merit.

**Comment by LoliconSlayer** [comment]
Tbh it really just sounds like a bunch of horse puns but whatever I guess

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
Thank you, the children are now safe from being demoralized. Today's world is filled with satanic influences. Can you remove devil's lettuce as well?

**Comment by boodaliboo** [comment]
I am here to save the children and what’s wrong with lettuce even if it’s not brown by devil don’t be racist

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm [hunk 1]
```diff
@@ -397,8 +397,8 @@
 	foodtype = FRUIT
 
 /obj/item/reagent_containers/food/drinks/bottle/applejack
-	name = "Buckin' Bronco's Applejack"
-	desc = "Kicks like a horse, tastes like an apple!"
+	name = "Farmer Johns All American Applejack"
```
Comment (Ktlwjec1) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm:400]: Farmer John's

**Comment by alexkar598** [comment]
I'm sorry I can't let this through, the power of friendship compels me to close this

**Pull request closed without merging**